src/Components/TaskForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react'
+
```

Comment:
I suggest removing this line.

Reply:
Suggestion resolved in the commits above